internal/campaigns/command.go

```diff
@@ -0,0 +1,18 @@
+package campaigns
+
+import (
+	"context"
+	"os/exec"
+)
+
+type OnCommand func(*exec.Cmd)
```

Comment:
Nitpick/not-a-blocker: I think `CommandWrapper` might be a better name here, since it's not a callback (which I always associate with "on" naming, which makes me think of events), but it does take in the command and execute it.